src/Sylius/Behat/Context/Setup/CurrencyContext.php

```diff
@@ -64,4 +73,35 @@ public function defaultCurrencyIs($currencyCode)
 
         $this->currencyRepository->add($currency);
     }
+
+    /**
+     * @Given the store has currency :currencyName
+     */
+    public function theStoreHasCurrency($currencyName)
+    {
+        $this->createCurrency($currencyName);
+    }
+
+    /**
+     * @Given the store has disabled currency :currencyName
+     */
+    public function theStoreHasDisabledCurrency($currencyName)
+    {
+        $this->createCurrency($currencyName, 'disabled');
+    }
+
+    /**
+     * @param string $currencyName
+     * @param string $currencyStatus
+     */
+    private function createCurrency($currencyName, $currencyStatus = 'enabled')
```

Comment:
Wdyt about ``$enabled = true``? We wouldn't have to check identity on string, and following ``if`` would be much nicer ;)